packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -202,20 +205,31 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
   }
 
   private _getStickyPlaceholderHeight(isSticky: boolean): React.CSSProperties {
-    const height = this.nonStickyContent ? this.nonStickyContent.offsetHeight : 0;
-
+    let height = 0,
+      width = 0;
+    if (this.nonStickyContent) {
+      height = this.nonStickyContent.offsetHeight;
+      width = this.nonStickyContent.scrollWidth;
+    }
     return {
       visibility: isSticky ? 'hidden' : 'visible',
-      height: isSticky ? 0 : height
+      height: isSticky ? 0 : height,
+      width: isSticky ? 0 : width
     };
   }
 
   private _getNonStickyPlaceholderHeight(): React.CSSProperties {
     const { isStickyTop, isStickyBottom } = this.state;
     if (isStickyTop || isStickyBottom) {
-      const height = this.nonStickyContent ? this.nonStickyContent.offsetHeight : 0;
+      let height = 0,
```

Comment:
Same as above:

`const height = this.nonStickyContent ? this.nonStickyContent.offsetHeight : 0;`
`const width = this.nonStickyContent ? this.nonStickyContent.scrollWidth : 0;`